_pages/dev/session-management.md

```diff
@@ -206,16 +205,20 @@ Once these settings are in place, Spartacus will use the defined flow out of the
 
-**Note:** The default OAuth server that is provided with SAP Commerce Cloud has functional but simplistic support for the Authorization Code Flow and the Implicit Flow (there is no way to log out a user from an external application, and no way to customize the login page).  For OCC versions 2211.X, it is expected that everyone using this OAuth server will continue to work with the Resource Owner Password Flow unless you use a different OAuth server (such as `Auth0`), you can switch to either of these flows.  With OCC version 2211-jdk21.0 and above, Authorization Code flow is the only flow available and it is expected to use the Custom Login Page feature to cover the shortcomings of the authorization server login form.
+**Note:** The default OAuth server that is provided with SAP Commerce Cloud has functional but simplistic support for the Authorization Code Flow and the Implicit Flow (there is no way to log out a user from an external application, and no way to customize the login page). If you are using a 2211.xx version of SAP Commerce Cloud (such as 2211.44, as opposed to 2211-jdk21.1), and you are using the OAuth server provided with SAP Commerce Cloud, it is expected that you will continue to work with the Resource Owner Password Flow. However, if you use a different OAuth server (such as `Auth0`), you can switch to either the Authorization Code Flow or the Implicit Flow. If instead, you are using a 2211-jdk21.x version of SAP Commerce Cloud (such as 2211-jdk21.1), the Authorization Code Flow is the only flow that is available, and it is expected that you will use the Custom Login Page feature (described in the next section) to cover the shortcomings of the authorization server login form.
+
+**Note:** If you are using a version of Spartacus that is older than 221121.1 (in other words, 2211.43 or older), ASM login only works with the Resource Owner Password Flow for customer support agents.  
+
+## Custom Login Page in Spartacus
 
-**Note:** Prior to 221121.0, ASM login only works with the Resource Owner Password Flow for customer support agents.  
+**Note:** This feature requires SAP Commerce Cloud version `2211-jdk21.1` or newer. It is not supported by 2211.xx versions of SAP Commerce Cloud that still supports JDK 17, such as SAP Commerce Cloud version 2211.44. Also, this feature is only relevant if you are using the authorization server provided by SAP Commerce Cloud 2211-jdk21.1 or newer. If you are using a different OAuth provider, you can ignore this feature or disable it.
 
-## Custom Login Page
+In Spartacus 221121.1 and newer, the Authorization Code Flow allows you to use the Spartacus login page instead of the authorization server login page. This helps ensure the branding and design of your site remain consistent during the login process.
```

Comment:
I would prefer to say that it's CCv2 that introduces this adjustment to Authorization Code flow, and Spartacus supports the flow change.  This sounds to me like Spartacus is the source of the flow change and implies we would enable custom login page to work with other oAuth solutions.